specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2021-11-15/restorable.json

```diff
@@ -0,0 +1,1815 @@
+          },
+          {
+            "in": "body",
+            "name": "location",
+            "required": true,
+            "description": "The name of the continuous backup restore location.",
+            "schema": {
+              "$ref": "#/definitions/ContinuousBackupRestoreLocation"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Table retrieve backup information operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/BackupInformation"
+            }
+          },
+          "202": {
+            "description": "The Table retrieve backup information operation will complete asynchronously."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/locations/{location}/restorableDatabaseAccounts/{instanceId}/restorableTables": {
+      "get": {
+        "operationId": "RestorableTables_List",
+        "x-ms-examples": {
+          "CosmosDBRestorableTableList": {
+            "$ref": "./examples/CosmosDBRestorableTableList.json"
+          }
+        },
+        "description": "Show the event feed of all mutations done on all the Azure Cosmos DB Tables. This helps in scenario where table was accidentally deleted. This API requires 'Microsoft.DocumentDB/locations/restorableDatabaseAccounts/.../read' permission",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/locationParameter"
+          },
+          {
+            "$ref": "#/parameters/instanceIdParameter"
+          },
+          {
+            "$ref": "#/parameters/restoreStartTime"
+          },
+          {
+            "$ref": "#/parameters/restoreEndTime"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/RestorableTablesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/locations/{location}/restorableDatabaseAccounts/{instanceId}/restorableTableResources": {
```

Comment:
>restorableTableResources [](http://example.com/codeflow?start=129&length=24)

Same comment for modeling as POST.

Reply:
Hi Jin,
We have similar API already existing for Sql/MongoDB (https://github.com/Azure/azure-rest-api-specs/blob/569bb5b5926008180266346d4af44647363446d4/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2021-11-15/restorable.json#L305).

We are just adding support for Tables and Gremlin for same restorableResources Api.